packages/sanity/src/_internal/cli/commands/schema/deploySchemaCommand.ts

```diff
@@ -1,6 +1,6 @@
 import {type CliCommandDefinition} from '@sanity/cli'
```

Comment:
This file in particular needs review: It should only use "deploy" vocabulary, not "store".